package.json

```diff
@@ -33,8 +34,9 @@
   },
   "devDependencies": {
     "level-mem": "^0.18.0",
-    "noddity-butler": "^2.1.1",
+    "noddity-butler": "~2.3.0",
     "noddity-linkifier": "^2.1.0",
-    "tape": "^4.2.0"
+    "tape": "^4.2.0",
```

Comment:
nah, `tape-catch` doesn't specify a dependency on `tape`, you have to do that yourself.  It just requires whatever version you have installed.